jb-lib/lib/channel_handler/__init__.py

```diff
@@ -0,0 +1,2 @@
+from lib.channel_handler.channel_handler import ChannelHandler, ChannelData, User
+from lib.channel_handler.pinnacle_whatsapp_handler import PinnacleWhatsappHandler
```

Comment:
Import telegram Handler here

Reply:
Convert lib.channel_handler to . convention

Reply:
done